docs/content/topics/developers.md

```diff
@@ -8,65 +8,67 @@ aliases:
+- git-initializer: The code that runs as a sidecar to fetch Git repositories
+  for vcs-enabled projects
 
-Additionally, there are several opt-in gateways that can be enabled via Helm chart values.  These are:
+This document covers environment setup, how to run tests and development of
+core components.
 
- - Brigade GitHub App Gateway: The implementation of the GitHub App web hooks. It requires
-  the controller.
- - Generic Gateway: A generic gateway offering flexibility to create Brigade events from webhooks
-  originating from an arbitrary service/platform.
-- Container Registry Gateway: A gateway supporting container registry webhooks such as the ones emitted by
-DockerHub and ACR.
+## Prerequisites
 
-Read up on all the gateways above, as well as others, in the [Gateways doc](./gateways.md).
+- A local Kubernetes cluster, 1.16.0+.  We recommend [kind] or [minikube].
+- [Docker] _or_ [Go 1.16+] and JS dependencies
+- make
 
-This document covers environment setup, how to run functional tests and development of
-`brigade-controller` and `brigade-worker` components.
+[kind]: https://github.com/kubernetes-sigs/kind
+[minikube]: https://github.com/kubernetes/minikube
+[Docker]: https://www.docker.com/
 
-## Prerequisites
+## Clone the Repository
 
-- Minikube or [kind](https://github.com/kubernetes-sigs/kind) (running k8s versions 1.16.0+)
-- Docker
-- make
+The first step to begin developing on Brigade is to clone the repo locally,
+if you haven't already done so.  First navigate to your preferred work
+directory and then issue the clone command:
+
+```console
+# Clone via ssh:
+$ git clone git@github.com:brigadecore/brigade.git
 
-## Clone the Repository In GOPATH
+# Or, via https:
+$ git clone https://github.com/brigadecore/brigade.git
+```
 
-Building from source does not _require_ code to be on your `GOPATH` since all
-builds are containerized by default, however, if you do have Go installed
-locally and wish (for instance) for your text editor or IDE's Go support to work
-properly with this project, then follow these optional steps for cloning the
-Brigade repository into your `GOPATH`:
+**Note**: this leaves you at the tip of **master** in the repository. As of
+writing, active development on Brigade v2 is happening on the `v2` branch. To
+switch branches, run the following command:
 
 ```console
-$ export GOPATH=$(go env GOPATH) # GOPATH is set to $HOME/go by default
-$ export PATH=$GOPATH/bin:$PATH
-$ mkdir -p $GOPATH/src/github.com/brigadecore
-$ git clone https://github.com/brigadecore/brigade $GOPATH/src/github.com/brigadecore/brigade
-$ cd $GOPATH/src/github.com/brigadecore/brigade
+$ git checkout v2
 ```
 
-**Note**: this leaves you at the tip of **master** in the repository where active development
-is happening. You might prefer to checkout the most recent stable tag:
+You might prefer to checkout the most recent stable tag:
```

Comment:
I can see the usefulness of this suggestion for people who's intention may be to build the latest release from source, but I also worry that this could lead to people submitting PRs that have a lot of merge conflicts with the tip of `v2`.

Any thoughts about just dropping this part?

Reply:
This above is my only remaining feedback. Premptively LGTM'ing. Dealer's choice on whether/how to address this comment.

Reply:
Good point; will drop.  Perhaps we can entertain adding again when we have stable 2.0.x tags out there (sans `beta`, etc.)